mod/da/pkg/store/store.go

```
@@ -117,3 +166,67 @@
+// GetBlobsFromStore returns all blob sidecars for a given slot.
+func (s *Store[BeaconBlockT]) GetBlobsFromStore(
+	slot math.Slot,
+) (*types.BlobSidecars, error) {
+	// Get the commitment list for this slot
+	serializedCommitments, err := s.IndexDB.Get(slot.Unwrap(), []byte(SLOT_COMMITMENTS_KEY))
+	if err != nil {
+		return &types.BlobSidecars{Sidecars: make([]*types.BlobSidecar, 0)}, nil // Return empty if not found
+	}
+
+	slotCommitments := &types.SlotCommitments{}
+	if err := slotCommitments.UnmarshalSSZ(serializedCommitments); err != nil {
```

golangci-lint

shadow: declaration of "err" shadows declaration at line 175

(govet)